core/src/main/resources/error/error-classes.json

```diff
@@ -63,5 +63,67 @@
   "WRITING_JOB_ABORTED" : {
     "message" : [ "Writing job aborted" ],
     "sqlState" : "40000"
+  },
+  "UNSUPPORTED_CHANGE_COLUMN" : {
+    "message" : [ "Please add an implementation for a column change here" ],
+    "sqlState" : "46110"
```

Comment:
46110 is not in the SQLSTATE standard; I believe that `0A000` is the standard SQLSTATE for unsupported operations.

Reply:
done